docs/guides/initialize-state-with-props.md

```diff
@@ -0,0 +1,126 @@
+---
+title: initialize-state-with-props
+# nav: N
+---
+
+In cases that model [dependency injection](https://en.wikipedia.org/wiki/Dependency_injection), such as when a store should be initialized with props from a component, the recommended approach is to use a vanilla store with React.context.
+
+> **Note**: *Because the store returned by* `create` *is a hook, passing it to a context provider may violate the* [Rules of Hooks](https://reactjs.org/docs/hooks-rules.html).
```

Comment:
Is it possible to remove the quote block here? I don't think its adding much and the website styles look to be struggling when combined with `>` + `****`

Reply:
no problem. `>` + `****` was from the v3 context doc